backend/src/test/kotlin/org/loculus/backend/controller/submission/GetReleasedDataEndpointTest.kt

```diff
@@ -53,11 +56,17 @@ class GetReleasedDataEndpointTest(
     val currentYear = Clock.System.now().toLocalDateTime(TimeZone.UTC).year
 
     @Test
-    fun `GIVEN no sequence entries in database THEN returns empty response`() {
+    fun `GIVEN no sequence entries in database THEN returns empty response & etag in header`() {
         val response = submissionControllerClient.getReleasedData()
 
         val responseBody = response.expectNdjsonAndGetContent<ProcessedData<GeneticSequence>>()
         assertThat(responseBody, `is`(emptyList()))
+        response.andExpect(status().isOk)
+            .andExpect(header().exists(ETAG))
+            .andExpect { result ->
+                val etag = result.response.getHeader(ETAG)
+                assertThat(etag, `is`(notNullValue()))
+            }
```

Comment:
That would be a more intuitive order of calls:

```suggestion
        val response = submissionControllerClient.getReleasedData()
            .andExpect(status().isOk)
            .andExpect(header().string(ETAG, notNullValue()))

        val responseBody = response.expectNdjsonAndGetContent<ProcessedData<GeneticSequence>>()
        assertThat(responseBody, `is`(emptyList()))
```

Reply:
@anna-parker

Reply:
this is also outdated - resolving